modules/serverless-cluster-sizing-requirements.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// serverless/installing-openshift-serverless.adoc
+
+
+[id="cluster-size-requirements_context"]
+= Cluster size requirements
+
+Determine how many nodes and pods you require for your OpenShift Container Platform cluster. Cluster scalability correlates to the number of pods in a cluster environment. That number influences the other numbers in your setup.
+
+// Scalability, planning requirements, and limits
+include:scalability_and_performance/planning-your-environment-according-to-object-limits.adoc[leveloffset=+1]
+
+Your cluster must be sized appropriately to ensure the instructions work properly. To do so, you can use the MachineSet API to manually scale your cluster up to the desired size.
```

Comment:
We should add some details about what MachineSet API is or a link to existing docs etc; to me it's confusing to just mention this but not have a procedure or some explanation?